client/packages/system/src/Report/api/hooks/utils/usePrintReport.ts

```diff
@@ -41,7 +41,14 @@ const printPage = (url: string) => {
         if (frame.contentDocument)
           frame.contentDocument.documentElement.innerHTML = html;
 
-        print(frame);
+        // Wait till images are loaded. There seem to be no callback for this and its hard to check
+        // if everything is loaded correctly for unknown html. For this reason, just wait a small
+        // amount of time.
+        // Note, while developing, even a timeout of 0ms worked but it has been raised o 30ms to
+        // have some buffer.
+        setTimeout(() => {
+          print(frame);
+        }, 30);
```

Comment:
is there some callback? onload didn't load because the frame was already loaded...

Reply:
In theory, a `MutationObserver`, though I would have expected that `readystatechange` or `DOMContentLoaded` would have fired. Nothing is working reliably for me though.
I think just stick with the timeout.. it's working